src/app/showcase/colors-showcase/colors-showcase.component.html

```diff
@@ -1,39 +1,47 @@
 <p>Below is the Kirby color palette. You can try out the different color combinations there and see how it can be used in the CSS in the code example box. All the colors are fetched from the <code>scss/themes/_colors.scss</code> file.</p>
 <div class="color-sample-wrapper">
-  <div class="code-wrapper">
-    <div class="kirby-text-large">Example use:</div>
-    <pre>
+  <div class="h2">Example use (Click on a color below to change it):</div>
+  <div class="color-sample-flex">
+    <div class="color-sample-example {{selectedColor}}">
+      <div class="color-sample-on-text {{selectedOnColor}}">
+        Contrast Text
+      </div>
+      <div class="color-sample-tint {{selectedColor}}-tint">Tint</div>
+      <div class="color-sample-shade {{selectedColor}}-shade">Shade</div>
+    </div>
+    <pre class="language-css">
 .color-sample-example {{'{'}}
   color: get-color('{{selectedOnColor}}');
   background-color: get-color('{{selectedColor}}');
+{{'}'}}
+.color-sample-tint {{'{'}}
+  background-color: get-color('{{selectedColor}}-tint');
+{{'}'}}
+.color-sample-shade {{'{'}}
+  background-color: get-color('{{selectedColor}}-shade');
 {{'}'}}</pre>
-    <p>The get-color function will output CSS variables for web and output the color values directly for Nativescript.</p>
-  </div>
-  <div class="color-sample-example {{selectedColor}}">
-    <div class="color-sample-on-text {{selectedOnColor}}">
-      Text
-    </div>
-  </div>
-  <div class="color-selector">
-    <div class="kirby-text-large">First pick color to set:</div>
-    <!-- TODO TRM - D. 21/12/2018 Radio buttons are currently unstyled because we have no design of them yet. -->
-    <p>
-      <input type="radio" id="color-bg" name="color-picker" [(ngModel)]="activeColorType" value="bg">
-      <label for="color-bg"> Background color</label>
-    </p>
-    <p>
-      <input type="radio" id="color-on" name="color-picker" [(ngModel)]="activeColorType" value="text">
-      <label for="color-on"> Text color</label>
-    </p>
   </div>
+  <p>The get-color function will output CSS variables for web and output the color values directly for Nativescript.</p>
```

Comment:
Consider wrapping method in pre tags:
```suggestion
  <p>The <pre>get-color</pre> function will output CSS variables for web and output the color values directly for Nativescript.</p>
```